lib/content_negotiation_filter.js

```diff
@@ -51,7 +94,8 @@ module.exports = (hyper, req, next, options, specInfo) => {
             return htmlRes;
         }
 
-        if (semver.gt(requestedVersion, storedVersion)) {
+        if (semver.gt(requestedVersion, storedVersion)
+                && !semver.satisfies(storedVersion)) {
```

Comment:
`!semver.satisfies(storedVersion)` needs a second argument

Reply:
Ouch, that's aremaining of my first idea how to fix that...